net-dialup/freeradius-client/freeradius-client-1.1.7-r1.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="FreeRADIUS Client framework"
+HOMEPAGE="https://wiki.freeradius.org/project/Radiusclient"
+SRC_URI="ftp://ftp.freeradius.org/pub/freeradius/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+
+IUSE="scp shadow static-libs"
+
+DEPEND="!net-dialup/radiusclient-ng"
+RDEPEND="${DEPEND}"
+
+DOCS="BUGS doc/ChangeLog doc/login.example doc/release-method.txt
+doc/instop.html README*"
+
+src_prepare() {
+	default
+	mv configure.in configure.ac || die \
+		"Renaming configure.in to configure.ac failed"
+	eautoreconf
+}
+
+src_configure() {
+	local myeconfargs=(
+		$(use_enable scp)
+		$(use_enable shadow)
+		--with-secure-path
+	)
+	econf "${myconf[@]}"
+}
+
+src_install() {
+	newdoc doc/README README.login.example
+	rm doc/README
+
+	default
```

Comment:
You call `default` first. `doc/README` is not matched in `DOCS` array, so it shouldn't get installed. Thus the `rm` call becomes obsolete.

Reply:
I’ve updated it, should be better now